android/src/main/java/proguard/dexfile/converter/Dex2Pro.java

```diff
@@ -419,7 +419,7 @@ private static Map<String, Clz> collectClzInfo(DexFileNode fileNode) {
                         switch (ann.type) {
                             case DexConstants.ANNOTATION_ENCLOSING_CLASS_TYPE: {
                                 DexType type = (DexType) findAnnotationAttribute(ann, "value");
-                                Clz enclosingClass = get(classes, type.desc);
+                                Clz enclosingClass = get(classes, type != null ? type.desc : null);
```

Comment:
Here and below, we should probably throw an explicit exception if the `type` or `m` is `null`. Since otherwise, it will look for a null key in the classes. It seems wrong that these would be null.

Or we could use `Objects.requireNonNull`.

```suggestion
                                Clz enclosingClass = get(classes, Objects.requireNonNull(type).desc);
```